index.html

```diff
@@ -1,92 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;500&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-	  <style>
-		  img {
-			  max-width: 100%;
-			  display: block;
-			  height: auto;
-			  margin: 0 auto;
-		  }
-		  #img-div {
-
-			  padding: 10px;
-			  margin: 0;
-		  }
-
-		  #main {
-			  margin: 30px 8px;
-			  padding: 15px;
-			  border-radius: 5px;
-			  background: #eee;
-		  }
-
-
-
-
-	  </style>
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TIAGOtube</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+<body>
+    <header>
+        <nav id= "nag-bar">
+            <div class="container">
+                <div class="bar1"></div>
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </div>
+            <a class="logo" href="#home">TIAGOtube &nbsp;&nbsp;&nbsp;&nbsp;</a>
+            <form id="search-box" action="http://thecodeblock.com">
+                <input  type="text" placeholder="Search..">
+                <button  type="submit" value="search">&nbsp;</button>
+                
+            </form>
+            <div>
+                
+            </div>
```

Comment:
Please don't leave empty div

Reply:
- Please always explain why the requested changes are required, so the student gets a better understanding or what is needed to change on his/her code.
For example
- Please don't leave empty div, so that your code look clean and professional